spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/main/java/com/alibaba/cloud/nacos/loadbalancer/NacosLoadBalancer.java

```diff
@@ -146,10 +146,7 @@ private Response<ServiceInstance> getInstanceResponse(
 				if (!CollectionUtils.isEmpty(sameClusterInstances)) {
 					instancesToChoose = sameClusterInstances;
```

Comment:
As you can see, we can change the cross-cluster log level information to, as this may happen normally, a warning level log printing information when the cluster call does not select the appropriate instance. Because no matter what the result is in the end, the instance will be selected. IMO, it is good to print more logs, which can help us debug better. 😆